Controller/home.py

```diff
@@ -41,10 +40,12 @@ def login():
 
 @home_route.route('/auth', methods=['GET'])
 def auth():
-    data = user.get_auth_user_dao(session['email'])
-    data["wishlist"] = application.get(session['email'], '')
-    return render_template('home.html', data=data, upcoming_events=upcoming_events)
-
+    if 'email' in session: 
+        data = user.get_auth_user_dao(session['email'])
+        data["wishlist"] = application.get(session['email'], '')
```

Comment:
I dont think it just populates the wishlist, it gets all the applications of the user, can you please create a bug to change the name here.

Reply:
Yes. Will look into it